arch/arm/src/armv8-r/arm_gicv3.c

```diff
@@ -250,7 +250,7 @@ void arm_gic_irq_enable(unsigned int intid)
 
   if (GIC_IS_SPI(intid))
     {
-      arm_gic_write_irouter(up_cpu_index(), intid);
+      arm_gic_write_irouter(this_cpu(), intid);
```

Comment:
should be up_cpu_index, not this_cpu:
https://github.com/apache/nuttx/pull/14503